### ISSUE TITLE
v-model doesn't work

### ISSUE THREAD

**Issue by Yusov** [trevoreyre/autocomplete#128]
I can't use v-model. What happened?

**Comment by gburning** [comment]
I don't think this package is currently compatible with the v-model pattern. You can set a default value with `defaultValue` and then you'd listen for `submit` to get the value.

If you really need to set the value externally you can use the `autocomplete` component's `setValue` method:
https://github.com/trevoreyre/autocomplete/issues/25
